api/commands/build.ts

```diff
@@ -115,15 +115,13 @@ export class Builder {
       throw new Error("Protected datasets cannot be fully refreshed.");
     }
 
-    const tasks = table.disabled
-      ? ([] as dataform.IExecutionTask[])
-      : this.adapter.publishTasks(table, runConfig, tableMetadata).build();
-
     return {
       ...this.toPartialExecutionAction(table),
       type: "table",
       tableType: table.type,
-      tasks,
+      tasks: table.disabled
```

Comment:
This won't skip things like pushing metadata back to bigquery, but that's not the end of the world

Reply:
good catch, but actually the run code does correctly skip that code: https://source.cloud.google.com/tada-analytics/github_dataform-co_dataform/+/master:api/commands/run.ts;l=325?q=executeAction